### DISCUSSION TITLE
如何使用备份功能？

### DISCUSSION THREAD

**Discussion by wayner6** [usememos/memos#2005] [General]
看到最新版本更新了备份功能的ui，但是没有找到

**Comment by dianso** [comment]
设置时间，重启

**Comment by wayner6** [comment]
@dianso 设置时间是指？我重新拉取了docker镜像并重启，还是找不到备份功能

**Comment by imaegoo** [comment]
![image](https://github.com/usememos/memos/assets/20182252/86bbc199-a2b8-4f1b-8b7e-85e8093f179b)

自动备份功能会定时备份 `memos_prod.db` 这个文件，备份方式是原地复制一份，文件名以时间结尾

（<https://github.com/usememos/memos/pull/1950>）原文：

> The backup files will be named as the db file with a suffix, like -YYYmmdd-hhmmss.

**Comment by wayner6** [comment]
好的，谢谢，明白了。  
我以为可以备份到外面，感觉原地复制一份是不是太过于鸡肋了？一般都是服务器出故障了才会用到备份功能，这时可能都进不去备份目录，是否考虑采用webdav等方式进行备份

**Reply by imaegoo** [reply]
我理解该功能应该是为防止db文件损坏而设计的。

如果想做webdav定时备份，我觉得直接在服务器上加一个定时任务就行。

当然如果memos能直接支持就更好了。

**Reply by wayner6** [reply]
好的
